pallets/kitties/src/lib.rs

```diff
@@ -341,6 +342,9 @@ pub mod pallet {
 			let new_cnt = Self::kitty_cnt().checked_add(1)
 				.ok_or(<Error<T>>::KittyCntOverflow)?;
 
+			// Check if the kitty does not already exist in our storage map
+			ensure!(Self::kitties(&kitty_id) == None, <Error<T>>::KittyExists);
```

Comment:
```suggestion
			ensure!(Self::kitties(&kitty_id) == None, <Error<T>>::KittyNotExist);
```
Will put this in a separate commit. Didn't catch this in time.

Reply:
This should be `KittyExists`, not `KittyNotExist`. Reason being, the user is attempting to create a kitty that already exists, which is thrown as an error `KittyExists`.